pkg/volume/gce_pd/gce_util.go

```diff
@@ -261,39 +270,73 @@ func createRegionalPD(
 }
 
 // Returns the first path that exists, or empty string if none exist.
-func verifyDevicePath(devicePaths []string, sdBeforeSet sets.String) (string, error) {
+func verifyDevicePath(devicePaths []string, sdBeforeSet sets.String, diskName string) (string, error) {
```

Comment:
@msau42 is this method going to be refactored out as part of the mount utilities refactor?

Also can this be made into a public interface?

Reply:
It's not being considered as part of the Mount utils refactor, but we can look into adding this there after the initial refactor is done.  Right now, only GCE PD does this kind of checking.